kedro-datasets/setup.py

```diff
@@ -84,6 +84,7 @@ def _collect_requirements(requires):
     ]
 }
 video_require = {"video.VideoDataSet": ["opencv-python~=4.5.5.64"]}
+xarray_require = {"xarray.GeoTIFFDataSet": ["rioxarray>=0.15.0"]}
```

Comment:
rioxarray 0.14.0+ dropped support of python 3.8 (https://corteva.github.io/rioxarray/stable/history.html#id2)

Reply:
kedro-datasets will follow NEP 29, hence you can focus on Python 3.9+